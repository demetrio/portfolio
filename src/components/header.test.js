import React from 'react';
import Header from './header';
import { render } from '@testing-library/react';

describe('Header', () => {
	it('renders correctly', () => {
		const { container } = render(<Header siteTitle="Default Starter" />);
		expect(container).toMatchSnapshot();
		expect(container).toBeDefined();
	});
});
