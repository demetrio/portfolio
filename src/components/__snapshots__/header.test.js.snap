// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <header
    class="css-hnjfqn"
  >
    <a
      class="title css-1dtxhj1"
      font-weight="bold"
      href="/"
    >
      Default Starter
    </a>
    <a
      class="css-1ed1fe6"
      href="/"
    >
      <img
        alt="avatar"
        class="logo"
        src="test-file-stub"
      />
    </a>
    <nav
      css="You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop)."
    >
      <a
        class="css-6uh4pm"
        href="/"
      >
        Home
      </a>
      <a
        class="css-6uh4pm"
        href="/about/"
      >
        About
      </a>
      <a
        class="css-6uh4pm"
        href="/projects/"
      >
        Portfolio
      </a>
    </nav>
  </header>
</div>
`;
